install/tools/man/ipa-idrange-fix.1

```diff
@@ -0,0 +1,111 @@
+you are consenting to the tool's propositions.
+
+You can apply the propositions manually via \fIipa idrange(1)\fR commands.
+
+This tool requires to be run from \fBroot\fR, and does not require a kerberos
+ticket. The directory server needs to be running.
+
+\fIipa-idrange-fix\fR will read currently set ranges from LDAP, then check
+their basic constraints, RID bases, etc. If it finds critical issues with
+ranges, manual adjustment will be required.
+
+After analyzing already present ranges, the tool will search for users and 
+groups that are outside of ipa-local ranges, then it will attempt to propose
+new ipa-local ranges in order to cover users and groups found.
+
+Finally, the tool will summarize the analysis, and, if there are proposed
+changes, will ask if the user wants to apply those. Please read the
+propositions carefully before proceeding with changes!
+
+Important note: By default, \fIipa-idrange-fix\fR will not cover the users and 
+groups that have IDs under 1000, as these IDs are reserved for system and
+service users and groups. We \fBdon't recommend\fR using IDs under 1000 for
+IPA users and groups, as they can possibly overlap with local ones. Please
+consider moving those users out of the range 1..1000, unless you absolutely
+have to.
+
+.SH "OPTIONS"
+.TP
+\fB\-\-version\fR
+Show the program's version and exit.
+.TP
+\fB\-h\fR, \fB\-\-help\fR
+Show the help for this program.
+.TP
+\fB\-\-ridoffset \fIINT\fR
+An offset tool is using to propose new base RIDs for ranges. We introduce 
+offset in order to have an ability to increase ranges in the future, increase 
+to more than offset will result to RID bases overlapping, and will be denied. 
+If set to 0, there will be no offset, proposed RID ranges will start directly 
+one after another.
+
+Default - \fI100000\fR, allowed values - from \fI0\fR to \fI2^31\fR.
+.TP
+\fB\-\-rangegap \fIINT\fR
+A number of IDs between out of ranges IDs to be considered to big to be inside 
+a proposed range. If the gap is bigger than this attribute, new range will be 
+started. If set to 0, every entity will get it's own range, if allowed by 
+\fI--minrange\fR.
+
+Default - \fI200000\fR, allowed values - from \fI0\fR to \fI2^31\fR.
+.TP
+\fB\-\-minrange \fIINT\fR
+A minimal size of IDs in a range the tool considers to be a valid range. All 
+IDs in ranges with less than this number will be considered outliers, not 
+worth creating an IDrange for, and will be listed explicitly to be moved 
+manually. If set to 1, every entity, even if single in the middle of an empty 
+space, will be proposed with a range.
+
+Default - \fI10\fR, allowed values - from \fI1\fR to \fI2^31\fR.
+.TP
+\fB\-\-allowunder1000\fR
+A flag to allow proposing ranges that start with IDs lower than \fI1000\fR. 
+Remember, this is not recommended - IDs under 1000 are reserved for system and 
+service users and groups, and IDranges with these low IDs may result with 
```

Comment:
end the setence at groups: "and groups. IDranges with these..."

Reply:
fixed